app/models/user.rb

```diff
@@ -0,0 +1,28 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  validates :phone_number, length: {is: 11}
+  validate :password_lower_case
+  validate :password_uppercase
+  validate :password_special_char
+
+  def password_uppercase
+    return if !!password.match(/\p{Upper}/)
+    errors.add :password, ' must contain at least 1 uppercase '
```

Comment:
```suggestion
    errors.add :password, 'must contain at least 1 uppercase '
```
same for all methods below